package-lock.json

```diff
@@ -3260,6 +3274,14 @@
 				"magic-string": "^0.25.0"
 			}
 		},
+		"node_modules/@surma/rollup-plugin-off-main-thread/node_modules/magic-string": {
+			"version": "0.25.9",
```

Comment:
seems like this does actually add to the dep tree because you're using `0.30` while the existing one was `0.25`?

Reply:
Only affects us because we're using a rather ancient version in our lockfile; in an actual install, they'd resolve to the same version.

Do you prefer that I also bump the existing copy of `magic-string` in our lock file?

Reply:
Sorry if you saw the previous message, I realized I wasn't looking at what you had actually commented on.

Will take a peek, can probably drop the version used (& bump back to latest in that other PR)